terraform/modules/hub/files/tasks/frontend.json

```diff
@@ -111,9 +117,18 @@
     }, {
       "Name": "AB_TEST_FILE",
       "Value": "/verify-frontend/federation/configuration/ab_test/${ab_test_file}"
+    }, {
+      "Name": "ABC_VARIANTS_CONFIG",
+      "Value": "/verify-frontend/federation/configuration/special-cases/abc-variants.yml"
     }, {
       "Name": "SEGMENT_DEFINITIONS",
       "Value": "/verify-frontend/federation/configuration/segment_definitions.yml"
+    }, {
+      "Name": "SEGMENT_DEFINITIONS_VARIANT_B",
+      "Value": "/verify-frontend/federation/configuration/segment_definitions_variant_b.yml"
```

Comment:
I guess variant B could use the control segment definitions, since it'll use the same segment, just not all of them...

Reply:
True - although in stub that wasn't true - and so there's code to support different configurations in each of the 3 tests (control A, variant B, variant C).